thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detector/email/filter/TestAlertFilterUtil.java

```diff
@@ -15,27 +15,26 @@
   @Test
   public void testPrecisionAndRecall() throws Exception{
     AlertFilter dummyAlertFilter = new DummyAlertFilter();
-    AlertFilterEvaluationUtil evaluator = new AlertFilterEvaluationUtil(dummyAlertFilter);
     // test data with 1 positive feedback, 1 negative feedback, other NA feedbacks
     List<MergedAnomalyResultDTO> anomalies = getMockMergedAnomalies(7,8);
```

Comment:
We should probably rename the test class since the original class name is renamed